serial_driver:
  ros__parameters:
    port: "/dev/ttyACM0"
    baudrate: 115200
    tolerance: 0.5
    timeout_sec: 10.0
    read_frequency: 20.0

vision_node:
  ros__parameters:
    camera_offset_x: 0.0   # cm
    camera_offset_y: 15.0  # cm (Example: Camera is 15cm in front of base)
    pixels_per_cm: 12.0

planner_node:
  ros__parameters:
    pickup_height_mm: 50.0 # Z height to grab objects
    
    # Classification Zones (X, Y, Z in mm)
    # Zone 1: Onion
    zone_onion_x: 150.0
    zone_onion_y: 150.0
    zone_onion_z: 00.0
    
    # Zone 2: Garlic
    zone_garlic_x: -150.0
    zone_garlic_y: 150.0
    zone_garlic_z: 00.0

    # Home Position (Angles)
    home_angles: [0.0, 0.0, 00, 0.0]